lib/index.ts

```diff
@@ -87,16 +87,30 @@ export async function fetchBeaconByTime(client: ChainClient, time: number): Prom
 }
 
 // an async generator emitting beacons from the latest round onwards
```

Comment:
Is this comment still at the right place?

Reply:
have moved the options below